packages/webapp-libs/webapp-tenants/src/components/tenantForm/tenantForm.hook.ts

```diff
@@ -0,0 +1,24 @@
+import { useApiForm } from '@sb/webapp-api-client/hooks';
+import { useEffect } from 'react';
+
+import { TenantFormFields, TenantFormProps } from './tenantForm.component';
+
+type UseTenantFormProps = Omit<TenantFormProps, 'loading'>;
+
+export const useTenantForm = ({ error, onSubmit, initialData }: UseTenantFormProps) => {
```

Comment:
maybe that's just my impression but this hook looks pretty generic, I'm not sure if we really need to name it `useTenantForm`, perhaps it could be named a bit more "neutral" (and put it in the general utils/hooks folder)? But if it's something that is supposed to be used only for this specific form (at least for now) we can go with this name I guess and maybe refactor it later.

Reply:
Yeah, this is for now only for the purpose of this form only ;)